src/table/index.js

```diff
@@ -242,18 +242,28 @@ class TableField extends Component {
     let currentTableObj = Object.keys(currentTableData);
     let tableList = currentTableObj.map(function(item, i) {
       if (typeof currentTableData[item] === "object") {
+        let isComponentDataAvailable = false;
         let currentable = currentTableData[item];
-        let diagnosisList = Object.keys(currentTableData[item]);
+        let tableList = Object.keys(currentTableData[item]);
+        let tableListData = tableList.map(function(item, i) {
+          if (Object.keys(currentable[item]).length > 0) {
+            isComponentDataAvailable = true;
+            return (
+              <li>
+                {currentable[item].code + " - " + currentable[item].description}
```

Comment:
It is better to get this from the schema so that the same functionality can be used in other tables